Cargo.lock

```diff
@@ -654,9 +662,9 @@ dependencies = [
 
 [[package]]
 name = "chunked_transfer"
```

Comment:
cc @bmwill what do you think of this crate? Do we know the owner?

Reply:
No i don't know the author and don't have an opinion on the crate, seems like its needed by ureq